server/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.java

```diff
@@ -1072,20 +971,13 @@ else if (hasSpineTracker)
 	 * @return the rotated Quaternion
 	 */
 	private Quaternion extendedKneeYawRoll(Quaternion knee, Quaternion ankle) {
-		// Clone the knee since we're modifying it and returning it
-		knee = knee.clone();
-
-		// Get the inverse rotation of the knee
-		rotBuf3.set(knee).inverseLocal();
-
 		// R = InverseKnee * Ankle
 		// C = Quaternion(-R.x, 0, 0, R.w)
 		// Knee = Knee * R * C
 		// normalize(Knee)
-		rotBuf3.multLocal(ankle);
-		rotBuf4.set(-rotBuf3.getX(), 0, 0, rotBuf3.getW());
-		knee.multLocal(rotBuf3).multLocal(rotBuf4);
-		return knee.normalizeLocal();
+		Quaternion r = knee.inv().times(ankle);
+		Quaternion c = new Quaternion(r.getW(), -r.getX(), 0, 0);
```

Comment:
What's this line doing?

Reply:
Not sure what you mean? Same as before..

Reply:
Just curious about the math behind it, I know it's still the same, but I'm confused what its purpose really is

Reply:
You’d have to ask AxisAngle about that one, I’m not able to describe *why* it works…

Reply:
I'll resolve this and ask more generall on the Discord then, this is not relevant to this PR.